### ISSUE TITLE
conda install ultralytics returns an error as PackageNotFoundError

### ISSUE THREAD

**Issue by bit-scientist** [ultralytics/ultralytics#4275]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

Right after creating a clean conda env, running the command `conda install ultralytics` returns the following:
```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - ultralytics

Current channels:

  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

### Environment

OS: Ubuntu 20.04.1 LTS
Python: 3.9.17

### Minimal Reproducible Example

`conda install ultralytics`

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by Kayzwer** [comment]
@bit-scientist you also can get the project by `git clone https://github.com/ultralytics/ultralytics` and `cd ultralytics && pip install -e .`

**Comment by bit-scientist** [comment]
@Kayzwer thanks, I sure can. I have been using it for months. The reason why I posted this is not the fact that I can't install it, but letting the maintainers know the issue exists.

**Comment by glenn-jocher** [comment]
Thanks for bringing this to our attention, @bit-scientist. We sincerely appreciate users like you who take the time to notify us about potential issues.

Concerning the installation via conda, the Ultralytics suite, including YOLOv8, is not currently available as a conda package. That's why you receive a PackageNotFoundError when running `conda install ultralytics`. We distribute our software through pip and GitHub. 

You can clone our GitHub repo and install the necessary dependencies locally by using pip. We'll take your suggestion under advisement and consider whether distributing our software as a conda package would be beneficial for our users in the future.

Again, we really appreciate your feedback on this. It's feedback like yours that helps us improve the accessibility and usability of our software. Keep the suggestions coming!

**Comment by glenn-jocher** [comment]
@abraha2d it seems like conda install is failing for `conda install ultralytics`. I checked https://anaconda.org/conda-forge/ultralytics and everything seems fine.

The feedstock repo is automatically pushing the latest updates, I don't see any problems there either. Could you help us here?
https://github.com/conda-forge/ultralytics-feedstock/

**Comment by glenn-jocher** [comment]
@bit-scientist I just checked the anaconda page and it looks like you might need to add a `-c conda-forge` to the command, i.e.:
```bash
conda install -c conda-forge ultralytics
```

Can you try this command?

**Comment by glenn-jocher** [comment]
@bit-scientist I've created a PR to update the docs in #4272. Once you confirm this updated install command works I'll merge the PR and this should help all future users that are running into this issue.

**Comment by glenn-jocher** [comment]
@bit-scientist great news 😃! Your original issue may now be resolved ✅ in PR #4272. 

To get this update:
- **[Git](https://github.com/ultralytics/ultralytics)** – Run `git pull` from within your `ultralytics/` directory or run `git clone https://github.com/ultralytics/ultralytics` again
- **[Pip](https://pypi.org/project/ultralytics/)** – Update with `pip install -U ultralytics`
- **[Notebooks](https://github.com/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb)** – Check out the updated notebooks  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/ultralytics)** – Run `sudo docker pull ultralytics/ultralytics:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and letting us know. Please confirm if this update fixes the issue for you, and don't hesitate to report any other **issues** you find or **feature requests** you may have. Happy training with YOLOv8 🚀!
